File: src/main/java/io/github/oleksivio/telegram/bot/api/model/objects/inline/queryresult/TitledInlineResult.java
package io.github.oleksivio.telegram.bot.api.model.objects.inline.queryresult;

import com.fasterxml.jackson.annotation.JsonProperty;

public abstract class TitledInlineResult extends FormattedInlineResult {
    /**
     * title String Title for the result
     */
    @JsonProperty("title")
    private String title = null;

    TitledInlineResult(String type) {
        super(type);
    }

    public final String getTitle() {
        return title;
    }

    public final void setTitle(String title) {
        this.title = title;
    }
}

File: src/main/kotlin/io/github/oleksivio/telegram/bot/api/model/objects/inline/queryresult/TitledInlineResult.kt
package io.github.oleksivio.telegram.bot.api.model.objects.inline.queryresult

import com.fasterxml.jackson.annotation.JsonProperty

abstract class TitledInlineResult : FormattedInlineResult() {
    /**
     * title String Title for the result
     */
    @JsonProperty("title")
    var title: String? = null
}
